src/commands/synthetics/README.md

```diff
@@ -307,22 +222,245 @@ yarn datadog-ci synthetics run-tests -f ./component-1/**/*.synthetics.json -f ./
+
+Configuration options:
+
+* Global Config: `"batchTimeout": 180000`
+* ENV variable: `DATADOG_SYNTHETICS_BATCH_TIMEOUT=180000`
+* CLI param: `--batchTimeout 180000`
+
+#### `configPath`
+
+The global JSON configuration is used when launching tests. See the [example configuration](https://docs.datadoghq.com/continuous_testing/cicd_integrations/configuration/?tab=npm#global-configuration-file-options ) for more details.
+
+Configuration options:
+
+* Global Config: N/A
+* ENV variable: `DATADOG_SYNTHETICS_CONFIG_PATH=global-config.json`
+* CLI param: `--config global-config.json`
+
+#### `datadogSite`
+
+The Datadog instance to which request is sent. The default is `datadoghq.com`.<!-- partial Your Datadog site is {{< region-param key="dd_site" code="true" >}}. partial -->
+
+Configuration options:
+
+* Global Config: `"datadogSite": "datadoghq.com"`
+* ENV variable: `DATADOG_SITE=datadoghq.com`
+* CLI param: `--datadogSite datadoghq.com`
+
+#### `defaultTestOverrides`
+
+Overrides for Synthetic tests applied to all tests.
+
+Configuration options:
+
+* Global Config: See `testOverrides` part of [Test files](#test-files) for an example.
+* ENV variable: N/A
+* CLI param: N/A
+
+#### `failOnCriticalErrors`
+
+A boolean flag that fails the CI job if no tests were triggered, or results could not be fetched from Datadog. The default is set to `false`.
+
+Configuration options:
+
+* Global Config: `"failOnCriticalErrors": true`
+* ENV variable: `DATADOG_SYNTHETICS_FAIL_ON_CRITICAL_ERRORS=true`
+* CLI param: `--failOnCriticalErrors`
+
+#### `failOnMissingTests`
+
+A boolean flag that fails the CI job if at least one specified test with a public ID (a `--public-id` CLI argument or listed in a [test file](#test-files)) is missing in a run (for example, if it has been deleted programmatically or on the Datadog site). The default is set to `false`.
+
+Configuration options:
+
+* Global Config: `"failOnMissingTests": true`
+* ENV variable: `DATADOG_SYNTHETICS_FAIL_ON_MISSING_TESTS=true`
+* CLI param: `--failOnMissingTests`
+
+#### `failOnTimeout`
+
+A boolean flag that fails the CI job if at least one test exceeds the default test timeout. The default is set to `true`.
+
+Configuration options:
+
+* Global Config: `"failOnTimeout": true`
+* ENV variable: `DATADOG_SYNTHETICS_FAIL_ON_TIMEOUT=true`
+* CLI param: `--failOnTimeout`
+
+#### `files`
+
+Glob patterns to detect Synthetic test [configuration files](#test-files).
+
+Configuration options:
+
+* Global Config: `"files": ["{,!(node_modules)/**/}*.synthetics.json"]`
+* ENV variable: `DATADOG_SYNTHETICS_FILES="{,!(node_modules)/**/}*.synthetics.json"`
+* CLI param: `-f "{,!(node_modules)/**/}*.synthetics.json"` / `--files "{,!(node_modules)/**/}*.synthetics.json"`
+
+#### `jUnitReport`
+
+The filename for a JUnit report if you want to generate one.
+
+Configuration options:
+
+* Global Config: `"jUnitReport": "e2e-test-junit"`
```

Comment:
I just used the value used in the example in the reporter section
https://github.com/DataDog/datadog-ci/pull/1372/files#diff-5a0fda3a46d1c105f78a54a8d87902bdb9886b1739928f72abc7cefc23a2f86bR906
but i do like the idea of specifying that it will create an `.xml` file